### DISCUSSION TITLE
Hikari and ExceptionOverrideClassName

### DISCUSSION THREAD

**Discussion by marlongionazwift** [aws/aws-advanced-jdbc-wrapper#248] [General]
Hello!

In the example [here](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/examples/HikariExample/src/main/java/software/amazon/HikariExample.java) the "ExceptionOverrideClassName" in HikariConfig is not being set, and without that it was not working for me.

After seeing it being set in the Hikari integration test in the source code I added it and now it's working. 

Just wanted to confirm if this is really something that is missing in the example.

Regards,
Marlon

**Comment by karenc-bq** [comment]
Hi @marlongionazwift, thank you for raising this issue. You are correct, the example is missing this configuration. We will add it to the example.
Thanks again for pointing this out!
